doc/1/controllers/collection/update-specifications/snippets/update-specifications.test.yml

```diff
@@ -3,7 +3,7 @@ description: Update the validation specifications.
 hooks:
   before: |
     curl -X POST kuzzle:7512/nyc-open-data/_create
-    curl -X PUT kuzzle:7512/nyc-open-data/yellow-taxi
+    curl -X PUT kuzzle:7512/nyc-open-data/green-taxi
```

Comment:
I don't get that change, since the snippet seems to work on the `yellow-taxi` collection.

Reply:
Yes, I made a mistake, I didn't mean to change it.